isis/src/control/objs/BundleSettings/BundleSettings.cpp

```diff
@@ -11,7 +11,7 @@
 #include <QXmlInputSource>
 
 #include "BundleObservationSolveSettings.h"
-//#include "FileName.h"currently only used in commented code
+#include "FileName.h"
```

Comment:
Again, is this include needed? It looks like it was being used in commented-out code, but it does not look like that code was implemented...?

Reply:
No, that is not needed either.